app.py

```diff
@@ -80,7 +80,14 @@ def auth0_callback():
     request = app.current_request
     req_dict = request.to_dict()
     domain, context = get_domain_and_context(req_dict)
-    resp_headers = {'Location': context + 'view/' + DEFAULT_ENV}
+    # extract redir cookie
+    cookies = req_dict.get('headers', {}).get('cookie')
+    redir_url = None
```

Comment:
Maybe set `redir_url = context + 'view/' + DEFAULT_ENV` so that the homepage redirection works as a fallback?

Reply:
Good point. Done.